shin-core/src/layout/layouter.rs

```diff
@@ -446,15 +494,32 @@ pub fn layout_text(params: LayoutParams, text: &str) -> LayoutedMessage {
                 ParsedCommand::VoiceVolume(volume) => {
                     actions_builder.action(layouter.time, ActionType::VoiceVolume(volume))
                 }
-                ParsedCommand::Newline => layouter.on_newline(),
+                ParsedCommand::Newline => {
+                    // If character_name is true, finalise the character name part. Then set
+                    // character_name to false to signify that we are now processing the main message text
+                    // If it was false in the first place, just do a normal newline.
+                    if character_name {
+                        layouter.on_newline(false); // No line wrapping in the character name
+                        layouter.state.instant = false;
```

Comment:
should be true?

Reply:
Ah, it's because you are _finishing_ the character name. Maybe add a comment xD

Reply:
No, the newline ends the character name part, so the following text should not be displayed instantly. Although, I forgot to set `instant` to `true` at the start of the line :sweat_smile: I'll fix that

Reply:
(and github didn't show me your second comment at first... pushed another commit to add a comment here...)